wire-tests/src/test/proto-kotlin/com/squareup/wire/protos/kotlin/person/Person.kt

```diff
@@ -119,7 +119,7 @@ data class Person(
     data class PhoneNumber(
         @field:WireField(tag = 1, adapter = "com.squareup.wire.ProtoAdapter.STRING") val number:
                 String,
-        @field:WireField(tag = 2, adapter = "PhoneType.ADAPTER") val type: PhoneType =
+        @field:WireField(tag = 2, adapter = "PhoneType.ADAPTER") val type: PhoneType? =
```

Comment:
This type is optional in the proto, but was being generated as non-nullable. My refactoring ended up catching that and fixing it by accident :)

Reply:
Good catch!